resources/prep/git.md

```diff
@@ -96,7 +96,8 @@ Make sure your change appears online.
 git pull origin master
 ```
 
-```{admonition} "Remember `origin` is just a short name of the web address of the repository."
+```{admonition}
```

Comment:
It's missing an admonition argument (title), so the admonition doesn't show up. Here it is:

```suggestion
```{admonition} "Short names for repositories:"
```